packages/hash/external-services/postgres/postgresql.conf

```diff
@@ -1,6 +1,12 @@
 # Allow connections from all addresses
 listen_addresses = '*'
+log_min_messages = fatal
+
+# Modules
+shared_preload_libraries = 'wal2json'
 
 # Enable logical decoding for the wal2json extension
 wal_level = logical
-max_wal_senders = 1
+max_wal_senders = 2
```

Comment:
Not for the `max_wal_senders` in particular. I believe the default is 10, and 2 would allow you to manually create a replication slot in the DB in the dev environment, which I found useful for debugging. The other config was inspired by debezium's config and getting annoyed at log spam with the logging level